css-layout-classes/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>
+      CSS Layout Classes
+    </title>
+  </head>
+  <body>
+    <div class="container">
+
+      <h1>Choose your character!</h1>
+
+      <div class="row col-one-third">
```

Comment:
Elements should not be both a `row` and a `column`. You should have a `row` that contains `column` elements.